src/main/java/ca/nrc/cadc/nameresolver/Service.java

```diff
@@ -91,7 +91,7 @@
 
     // Queries the Sesame(only for vizier) service at CDS.
     VIZIER("vizier", "cdsweb.u-strasbg.fr", 80,
-            HttpMethod.GET, "/cgi-bin/nph-sesame/-o/V?",
+            HttpMethod.GET, "/cgi-bin/nph-sesame/-o/~V?",
             null, "HTTP/1.1\r\nHost: cdsweb.u-strasbg.fr\r\n\r\n", true);
```

Comment:
Good question.  I did it to truly see if something cached was broken and thought it might be better to leave it.  I'm not sure how many queries are done to VizieR every day.  I've set it back.